config/templates/chef-marketplace-cookbooks/non_root_solo.rb.erb

```diff
@@ -0,0 +1,8 @@
+cookbook_path '/opt/chef-marketplace/embedded/cookbooks'
+cache_path '/tmp/chef-marketplace/local-mode-cache'
```

Comment:
We could pre-create a directory in /var/opt/chef-marketplace for the non-root user and set perms on that accordingly and that would work, no?